internal/rbd/rbd_util.go

```diff
@@ -1675,6 +1674,18 @@ func (ri *rbdImage) RemoveMetadata(key string) error {
 	return image.RemoveMetadata(key)
 }
 
+// ListMetadata returns a map containing all metadata assigned to the RBD
+// image.
+func (ri *rbdImage) ListMetadata() (map[string]string, error) {
```

Comment:
is it used anywhere? if not please consider removing it.

Reply:
Sure! I just kept it reserving for future :-)